pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -1330,5 +1330,5 @@ func checkRegionsReplicated(c *gin.Context) {
 		c.String(http.StatusBadRequest, err.Error())
 		return
 	}
-	c.String(http.StatusOK, state)
+	c.IndentedJSON(http.StatusOK, state)
```

Comment:
What is the reason for this modification? Do we need to modify the corresponding API interface for the non-API mode?

Reply:
Avoid failure in JSON unmarshalling and maintain consistency with PD mode.